ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/HikariJdbcConnectionProvider.scala

```diff
@@ -14,6 +14,7 @@ import com.daml.platform.store.DbType
 import com.daml.platform.store.dao.HikariJdbcConnectionProvider._
 import com.daml.resources.ResourceOwner
 import com.zaxxer.hikari.{HikariConfig, HikariDataSource}
+import scalaz.concurrent.Task
```

Comment:
We're not using Scalaz here and I don't think it makes sense to pull it in for a single use.

Reply:
Perhaps use `RetryStrategy.exponentialBackoff` instead?